cross/boto/Makefile

```diff
@@ -0,0 +1,14 @@
+PKG_NAME = boto
+PKG_VERS = 2.38.0
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/$(PKG_NAME)/$(PKG_NAME)/archive
```

Comment:
PyPI the preferred location for Python projects if they're available there. Download Boto from here: https://pypi.python.org/packages/source/b/boto/boto-2.38.0.tar.gz.